src/odemis/acq/test/fastem_test.py

```diff
@@ -147,7 +147,7 @@ def tearDownClass(cls):
         pass
 
     def setUp(self):
-        self.descanner.physicalFlybackTime.value = 250e-6  # TODO why is this necessary??
+        self.descanner.physicalFlybackTime.value = 150e-6  # TODO why is this necessary??
```

Comment:
I just tested this, and if I remove this setUp, all tests still behave as expected and the flyback time is correctly set. So I think this is not necessary and you can remove the whole setUp.